app/services/proxy_status/dhcp.rb

```diff
@@ -0,0 +1,20 @@
+module ProxyStatus
+  class DHCP < Base
+
+    def subnets
+      return @subnets if @subnets
+
+      fetch_proxy_data do
+        @subnets = api.subnets.map do |subnet|
+          records = api.subnet(subnet['network'])
```

Comment:
@xprazak2 in the code above you are hitting two different api endpoints, first we fetch / (return a list of networks/subnets) and another one /network_id for each netwrok returned from the first query.

if we want to ensure this works with a large setup, we could do something puppet ca controller, which then can fetch / display each parameter at a time. we can later add a reservation controller, which can show each reservation / edit / destroy etc (similar to https://github.com/theforeman/foreman_dhcp_browser/blob/master/app/controllers/dhcp_controller.rb)

I don't think we need to change anything on the smart proxy api?